.github/workflows/build.yaml

```
@@ -45,3 +45,14 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  docs-build:
+    if: ${{ startsWith(github.ref, 'refs/heads/branch-') }}
+    needs: cpp-build
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/custom-job.yaml@cuda-118
```

Should this be using a different branch?
```suggestion
    uses: rapidsai/shared-action-workflows/.github/workflows/custom-job.yaml@branch-23.04
```